html/ui/html/pages/dgs.html

```diff
@@ -123,12 +123,12 @@ <h1 data-i18n="my_completed_orders">My Completed Orders</h1>
 <!--start Dividends+ code-->
 <div id="nsv_dividends_page" class="page">
   <section class="content-header">
-    <h1>Token Administration for Token Issuer</h1>
+    <h1>Asset Administration for Asset Issuer</h1>
```

Comment:
There are many others, so let me know if you can merge recent changes on yours first.